src/main/java/net/sf/mzmine/modules/visualization/ida/IDADataSet.java

```diff
@@ -273,7 +273,7 @@ public void setColor(int series, int item, Color c) {
      * 
      */
     public void highlightSpectra(double mz, double ppm, int minIntensity,
```

Comment:
Let me rephrase the question: why do you prefer to use only ppm tolerance in this case, when all other MZmine modules are using the MZTolerance class that can specify both absolute and relative tolerances?